.github/workflows/test.yml

```diff
@@ -16,15 +16,18 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-20.04  # newest available distribution, aka focal
+    runs-on: ubuntu-latest
```

Comment:
nit I think this undoes the value of knowing which dist it is..

Reply:
Noted!